src/backend/distributed/utils/maintenanced.c

```diff
@@ -248,7 +253,8 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 	{
 		int rc;
 		int latchFlags = WL_LATCH_SET | WL_TIMEOUT | WL_POSTMASTER_DEATH;
-		int timeout = 10000; /* wake up at least every so often */
+		double timeout = 10000.0; /* use this if the deadlock detection is disabled */
```

Comment:
Why is this a double? The timeout is specified as an int/long, so this doesn't seem right.